boards/arm/rak5010_nrf52840/rak5010_nrf52840_defconfig

```diff
@@ -0,0 +1,20 @@
+# SPDX-License-Identifier: Apache-2.0
+CONFIG_SOC_SERIES_NRF52X=y
+CONFIG_SOC_NRF52840_QIAA=y
+CONFIG_BOARD_RAK5010_NRF52840=y
+
+# Enable MPU
+CONFIG_ARM_MPU=y
```

Comment:
You could also enable by default the CONFIG_HW_STACK_PROTECTION, to catch stack overflows.
This consumes some resources, but most cortex-m board enable it.

Reply:
Ok I'll add it